### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
I would need to be told if there are misspellings or have any suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
+
+# FAQs:
+
+### What is Streamlink Portable?
+A program to build a portable version of Streamlink for Windows.
+
+### What are the supported platforms?
+Windows Vista SP2 or later with .NET Framework 4.5+.
+
+### What can i do with it?
```
Comment (gravyboat) [inline README.md]: Capital I here. (`What can I do with it?`)

File: README.md [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
+Build between three flavors:
+* Portable EXE (Recommended): For users who prefer to have all available files in the same folder.
+* Standalone EXE: For users who prefer to have all files packed within the executable.
+* Portable BAT: Same as Portable EXE but a batch file is generated instead of an executable.
+
+By default the building source is the <a href="https://github.com/streamlink/streamlink/commits/master">latest available commit</a> (nightly).
 <br>
--Open the Visual Studio Project inside the "Streamlink for Windows (Source code)" folder and compile it
+No system files are modified and you can run it from anywhere (such as flash or external drives).
```
Comment (gravyboat) [inline README.md]: Quick clarification here `(such as flash or external drives)` maybe make that `(such as flash drive or other external drives)`

File: README.md [hunk 3]
```diff
@@ -1,23 +1,38 @@
...
+
+### Can i download a ready-to-use release without using the program?
```
Comment (gravyboat) [inline README.md]: Capital I again here.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,38 @@
...
+Yes, you can find precompiled releases <a href="https://github.com/streamlink/streamlink-portable/releases">here</a>.
 <br>
--Copy the "Streamlink for Windows (Compiled)\Files" folder to "Streamlink for Windows (Source code)\bin\Debug" if needed
+Note that only stable releases are available, for nightly you have to use the builder.
```
Comment (gravyboat) [inline README.md]: `for nightly you have to use the builder.` to `for nightly builds you have to use the program.` the term `builder` is only used at one other point on line 2 and might be confusing as it is referred to as a program on line 27.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,38 @@
...
+# Download:
+
+You can get the latest git archive <a href="https://github.com/streamlink/streamlink-portable/archive/master.zip">here</a>.
 <br>
--Extract both folders from the ZIP file
+Inside the ZIP you will find two folders:
+* Streamlink for Windows (Compiled): The program executable, intended for end users.
+* Streamlink for Windows (Source code): The program source code, intended for developers.
+
+# FAQs:
```
Comment (gravyboat) [inline README.md]: This can just be `FAQ`

**Review by gravyboat — CHANGES_REQUESTED** [review]
Overall looks great! I added a few very minor notes but past those this looks excellent.

**Comment by RosadinTV** [comment]
@gravyboat Done, thanks for the review 😄

**Pull request merged**